tests/saw/spec/handshake/s2n_handshake_io.cry

```diff
@@ -465,9 +711,31 @@ valid_connection conn = valid_handshake (conn.handshake)
 
 // Tells if the connection struct is in a valid initial state
 initial_connection : connection -> Bit
-initial_connection conn = conn.handshake.handshake_type == INITIAL /\ conn.handshake.message_number == 0
+initial_connection conn =
+       conn.handshake.handshake_type < `(S2N_HANDSHAKES_COUNT)
+    /\ conn.handshake.message_number == 0
     /\ (conn.corked == 0 \/ conn.corked == 1)
-    
+
+       // The assumptions below specifically pertain to TLS 1.3 handshakes.
+       // Some of the logic that decides which TLS 1.3 handshake types are
+       // reachable has moved out of the functions that we're verifying, so
+       // we use the preconditions below instead. Issue
+       // https://github.com/aws/s2n-tls/issues/3047 is about consolidating
```

Comment:
https://github.com/aws/s2n-tls/issues/3047

Reply:
To be clear, are you requesting to abbreviate "`https://github.com/aws/s2n-tls/issues/3047`" to "`#3047`" in this comment? I've went ahead and done this, but let me know if you intended something else.

Reply:
Nope, sorry, I was just tagging the issue here for reference.

Reply:
Ah, my bad :)

I've reverted it back to the full `https://github.com/aws/s2n-tls/issues/3047` URL in the comments for consistency with the other URL references I've left.